sdk/servicebus/service-bus/test/stress/scenarioCloseOpen.ts

```diff
@@ -30,7 +30,7 @@ function sanitizeOptions(args: string[]): Required<ScenarioCloseOptions> {
     receiveBatchMaxWaitTimeInMs: options.receiveBatchMaxWaitTimeInMs || 10000,
     numberOfMessagesPerSend: options.numberOfMessagesPerSend || 1,
     delayBeforeCallingCloseInMs: options.delayBeforeCallingCloseInMs || 100,
-    shouldCreateNewClientEachTime: options.shouldCreateNewClientEachTime
+    shouldCreateNewClientEachTime: !!options.shouldCreateNewClientEachTime
```

Comment:
Same question again.

Reply:
Yeah, it's the type that's causing it. At runtime (I assume because of minimist?) it should be set but that's not what typescript is seeing based on our options definition.